### ISSUE TITLE
[A11Y] amp-image-lightbox: Needs an accessible (yet invisible) close button.

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#5503]
tapping the back-padding area around the image inside a `amp-image-lightbox` closes the lightbox but that is not accessible. There needs to be an invisible (but read by screen-reader, e.g. off-screen) button as the last element in `-amp-image-lightbox-container` that says `Close image lightbox`.

Closing the lightbox should move the focus back to the source image that opened the lightbox in the first place.

**Comment by aghassemi** [comment]
/to @jridgewell 
Please wait for https://github.com/ampproject/amphtml/pull/5542 and reuse.

**Comment by aghassemi** [comment]
Fixed by https://github.com/ampproject/amphtml/pull/5894
